package.json

```diff
@@ -26,6 +26,7 @@
     "lint": "npm run eslint && npm run stylelint",
     "prettier": "prettier --write '**'",
     "prettier-ci": "npm run prettier -- --list-different || (echo '\n\nThis failure means you did not run `yarn prettier` before committing\n\n' && exit 1)",
+    "prettier-dev": "pretty-quick --staged",
```

Comment:
Why `--staged`? The `--branch` mode sounds like more of what we would want since I think it will check all files in the pull request. This might be better than `--staged` since you won't have to remember to run the command before committing your files. In other words, you could safely run it at the _end_ of your work in the pull request.

Reply:
Good idea - I've switched over the cli arg to `--branch master` which should compare the current branches changes vs the master branch and apply prettier to any files that have changed.